.idea/modules.xml

```diff
@@ -2,8 +2,8 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/modules/app/Wnder.wnder.app.iml" filepath="$PROJECT_DIR$/.idea/modules/app/Wnder.wnder.app.iml" />
```

Comment:
This modification seems undesirable. Why did you do that ?

Reply:
I don't know either, I will change it next commit.